kubernetes-repositories.md

```diff
@@ -0,0 +1,106 @@
+
+#### Goals
+Create a broader base of repositories than the existing gh/kubernetes/kubernetes so that the project can scale. Present expectations about the centrality and importance of the repository in the Kubernetes ecosystem. Carries the endorsement of the Kubernetes community.
+
+#### Rules
+
+   * Must live under github.com/kubernetes/<project-name>
+   * Must adopt the Kubernetes Code of Conduct
+   * All code projects use the Apache Licence version 2.0. Documentation repositories should use the Creative Commons License version 4.0.
+   * Must adopt the CNCF CLA bot
+   * Must adopt all Kubernetes automation (e.g. /lgtm, etc)
+   * All OWNERS must be members of standing as defined by ability to vote in Kubernetes steering committee elections. in the Kubernetes community
+   * Repository must be approved by SIG-Architecture
+
+### FAQ
+
+*My project is currently in kubernetes-incubator, what is going to happen to it?*
+
+Nothing. We’ll grandfather existing projects and they can stay in the incubator org for as long as they want to. We expect/hope that most projects will either move out to ecosystem, or into SIG or Core repositories following the same approval process described below.
+
+
+
+*My project wants to graduate from incubator, how can it do that?*
+
+Either approval from a SIG to graduate to a SIG repository, or approval from SIG-Architecture to graduate into the core repository.
+
+
+
+*My incubator project wants to go GA, how can it do that?*
+
+For now, the project determines if and when it is GA. For the future, we may define a cross Kubernetes notion of GA for core and sig repositories, but that’s not in this proposal.
+
+
+
+*My project is currently in core, but doesn’t seem to fit these guidelines, what’s going to happen?*
+
+For now, nothing. Eventually, we may redistribute projects, but for now the goal is to adapt the process going forward, not re-legislate past decisions.
+
+
+
+*I’m starting a new project, what should I do?*
+
+Is this a SIG-sponsored project? If so, convince some SIG to host it, take it to the SIG mailing list, meeting and get consensus, then the SIG can create a repo for you in the SIG organization.
+
+
+
+Is this a small-group or personal project? If so, create a repository wherever you’d like, and make it an associated project.
+
+
+
+We suggest starting with the kubernetes-template-project to ensure you have the correct code of conduct, license, etc.
+
+
+
+*Much of the things needed (e.g. CLA Bot integration) is missing to support associated projects. Many things seem vague. Halp!*
+
+True, we need to improve these things. For now, do the best you can to conform to the spirit of the proposal (e.g. post the code of conduct, etc)
```

Comment:
Writing these requirements down is a good first step. Someone can take this as evidence or belief to drive actions towards doing the next steps of making the tooling adaptable for this purpose..